.github/workflows/slotSwapStgAPI.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       ARM_TENANT_ID: ${{ secrets.TERRAFORM_ARM_TENANT_ID }}
     steps:
       - uses: actions/checkout@v4
-      - uses: azure/login@v2
+      - uses: azure/login@v2.2.0
```

Comment:
Question for my own understanding... Is there a reason we want to specify the minor version in this change? It looks like `v2` tag without a minor version specified points to [v2.2.0](https://github.com/Azure/login/releases/tag/v2) 🤔

Reply:
I didn't realize that `v2` tag already pointed to `2.2.0`.  I'll remove the minor version so they are all consistent.